src/main.cpp

```diff
@@ -391,7 +392,7 @@ const int pawn_attacked[] = {S(-64, -14), S(-155, -142)};
 
         // For each piece type
         for (int p = 0; p < 6; ++p) {
```

Comment:
Why isn't this loop simply going from 1 to 7, avoiding all the ``p + 1``? I'd compute that once into a var for the eval array indexing, if only the avoid the eyebleed this is now.